electron/config.js

```diff
@@ -1,5 +1,36 @@
 import Store from "electron-store"
 
+let sortedLanguages = [
```

Comment:
Can we build this list from `src/editor/languages.js`?

Reply:
I tried to, but ended up having issues because the `editor` subdirectory belongs to the view, whereas the config is the electron side. Ideally, the way I would want it to work is to do a static generation at build time, like I did for the README/initial-content.

As for the text reverting back to plain text, I guess we could only append `-a` at the end of a new block if the default language is set to Plain Text.